src/components/Accordion/AccordionItem.tsx

```diff
@@ -26,15 +26,19 @@ export const AccordionItem: FC<Props> = ({
           }
         : item
     );
-  }
+  };
 
   return (
-    <li className="item">
-      <div className="item__header" onClick={handleCurrentItemClick}>
-        <h3 className="item__title">{item.title}</h3>
-        {currentItem === item ? <Minus /> : <Plus />}
+    <li className="accordion__item">
+      <div className="accordion__item--header" onClick={handleCurrentItemClick}>
+        <h3 className="accordion__item--title">{item.title}</h3>
```

Comment:
двойной дефис `--` используется для модификаторов в BEM. Если ты хочешь передать именно название, то достаточно одного `-`
```suggestion
      <div className="accordion__item-header" onClick={handleCurrentItemClick}>
        <h3 className="accordion__item-title">{item.title}</h3>
```